betagouv/preuve-covoiturage
# migration/fix global rule name

- [hotfix] import Commands in Kernel
- fix insee + - regression on staging
- fix(acquisition): async error fix
- fix registry.user perms to access trip list and stats
- fix(sql): activate migration
- fix(trip): fix paramters schema
- fix(trip): fix export data normalization
- deactivate secured cookie on for local environment
- Fix stats / trip list filter : send empty dates interval when these are defined then removed
- trip list : sql injection safe implementation if territory filter
- trip list : implement territory filter on stats (territory user bug) + territories array lateral fix
- fix lint issues
- fix territory migrations : + add territory renaming
- fix(trip): fix middleware + territory search
- fix(trip): export territory filter
- feat(trip): add policy filter
- fix(trip): clean up
- fix(trip): build export params
- fix(trip): fix schema
- Front : add validation rules
- Front : trip / stats filtre : make start date required and limit it to minus 1 Y
- fix lint
- use git tag/commit-number as sentry release name
- improve sentry event data for better display
- filter out 401 on RPC routes only
- front : trip / stats : campaign list : display only users campaign
- fix #1000 : trip list / starts error with territory user
- Front stats : carpoolers / car better calculation
- front : fix : current / outdated campaign filtering
- front : trip / stats filter fix : apply a default - 1 year and filter limit to - 2 Y
- fix : apply new distance range rule set to templates
- add user data to sentry on /v2/journeys
- fix : Apply territory campaign dashboard to admin campaign list
- Front : campaign form : Update min distance value on staggered mode when global distance range minum distance is changed
- front : Territory Admin can't create Territory Demo user
- lint fix : sentry config
- add git to aptfile
- test version
- test version
- test version
- test version
- fix Response
- test version
- add version to listen message
- test version
- set .env
- use CONTAINER_VERSION => git SHA as version
- [migration] add France in territories as country
- fx max-len
- remove activable/active for country
- fix cast_fkeys acquisition migration
- add Sentry reporting to Queues
- add territory:tree action
- fix empty application_id before migration
- fix : linking region to owns by mixing code dep and region insee
- fix: territory schema + policy tests
- add findByInseesTerritoryAction + tree conflict
- fix : front convert insee > to children territory ID + fix territory list lines duplicates
- AOM validation / visibility filter
- fix(territory): schema extra fields
- feat(migration): add migration command to sync region
- fix sync region dep, dep <> town based on insee (and not postcode)
- split sql files exec in migrations
- fix migration on production db
- add logging to sync:dep
- add logging to sync:dep
- disable targeted triggers in sync:dep
- set local apiUrl
- add sync:legacy command
- Migrate territory ui_status
- fix(schema): enforce active property
- fix activable.value get
- fix(territory): update get_territory_view_data + table
- fix territory front crud + schema alignment
- fix #1016 : align start date for stats and trip list to now() - 1Y at 00:00:00 local time
- fix #1016 : align start date for stats and trip list to now() - 1Y at 00:00:00 local time
- add check to template rules on policy:seed command
- fix max-len
- fix : no territories filter for territory user
- fix campaign admin filtering
- sentry integration
- Fix : territory user territory edit form (activable missing)
- add Beta gouv sentry DSN to non local environment
-  add Beta gouv sentry DSN to non local environment
-  add Beta gouv sentry DSN to non local environment
- lint fix
- change global_rules name max_trip_restriction. fix #914
